pydomcfg/tests/bathymetry.py

```diff
@@ -44,6 +45,60 @@ def flat(self, depth: float) -> Dataset:
         ds["Bathymetry"] = xr.full_like(ds["glamt"], depth)
         return _add_attributes(_add_mask(ds))
 
+    def channel(self, depth: float, stiff=1.0) -> Dataset:
+        """
+        Channel with seamount case.
+
+        Produces bathymetry of a channel with a Gaussian seamount in order to
+        simulate an idealised test case. Based on Marsaleix et al., 2009
+        doi:10.1016/j.ocemod.2009.06.011 Eq. 15.
+
+        Parameters
+        ----------
+        depth: float
+            Bottom depth (units: m).
+        stiff: float
+            Scale factor for steepness of seamount (units: None)
+
+        Returns
+        -------
+        Dataset
+        """
+        ds = self._coords
+
+        # Get horizontal grid
+        glamt, gphit = ds.variables["glamt"], ds.variables["gphit"]
+
+        # Find half way point for sea mount location
+        jpi, jpj = ds.dims["x"] // 2, ds.dims["y"] // 2
+        glamt_mid, gphit_mid = glamt[jpj, jpi], gphit[jpj, jpi]
+
+        # Define sea mount bathymetry
+        ds["Bathymetry"] = DataArray(
+            depth
+            * (
+                1.0
+                - 0.9
+                * np.exp(
+                    -(
+                        stiff
+                        / 40.0e3 ** 2
+                        * ((glamt - glamt_mid) ** 2 + (gphit - gphit_mid) ** 2)
+                    )
+                )
+            ),
+            dims=["y", "x"],
+        )
```

Comment:
At this point glamt and gphit are already `DataArray`. No need to re-initialize the DataArray, everything is altrady in good shape.
I.e.,
```python
ds["Bathymetry"]  = depth * ...
```

Reply:
Sounds good... I'm happy to glance through and show syntax and tricks! Tutorials are good, but I think this way you'll pick up the general idea behind `xarray` much faster.